pages/common/openssl.md

```diff
@@ -0,0 +1,23 @@
+# openssl
+
+> OpenSSL is a cryptography toolkit.
+
+- Check an SSL connection
+
+`openssl s_client -connect {{url}}:443`
```

Comment:
Suggestion: Change `{{url}}` to `{{domain}}`. A url would be `https://example.com/testing?foo=bar....`, but I think that openssl takes something more like simply `example.com`.